online-shop/src/app/navigation-bar/navigation-bar.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Please make sure you remove the spec files if you're not going to test your component.